cost/aws/object_storage_optimization/README.md

```diff
@@ -0,0 +1,65 @@
+## AWS Object Storage Optimization
+ 
+### What it does
+
+This Policy checks S3 buckets for older objects and can move old object to 'glacier' or 'deep archive' after a given period of time. The user can choose to delete old object by enabling 'delete action' option as mentioned in Enable delete action section below.
+
+### Functional Details
+ 
+- This policy identifies all S3 objects last updated outside of the specified timeframe
+- For all objects identified as old, the user can choose to move the object to Glacier or Glacier Deep Archive after the specified timeframe.
+ 
+### Input Parameters
+ 
+This policy has the following input parameters required when launching the policy.
+
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
+- *Move to Glacier after days last modified* - leave blank to skip moving
+- *Move to Glacier Deep Archive after days last modified* - leave blank to skip moving
+- *Exclude Tag* - exclude object with the included tags 
+ 
+### Required RightScale Roles
+ 
+- Cloud Management - The `credential_viewer`,`observer` roles
+- Cloud Management - The `policy_designer`, `policy_manager` & `policy_publisher` roles
+
+### Enable delete action
+
+Perform below steps to enable delete action.
+
+- Edit the file [AWS_Unencrypted_RDS_Instances](https://github.com/flexera/policy_templates/tree/master/cost/aws/object_storage_optimization/aws_object_storage_optimization.pt)
+- uncomment the line which conatins 'escalate $esc_delete_s3_objects_approval' and save the changes.
```

Comment:
There should be a line that also says to make a comment on the **escalate $esc_modify_s3_object_storage_class_approval**

Reply:
@cdwilhelm 
I had mentioned comment statement in .pt file. I will add here as well, thanks.